sigal/gallery.py

```diff
@@ -179,8 +179,9 @@ def __init__(self, filename, path, settings):
         base = splitext(filename)[0]
         self.date = None
         self.src_filename = filename
-        self.filename = self.url = base + '.webm'
-        self.dst_path = join(settings['destination'], path, base + '.webm')
+        if not settings['use_orig']:
+            self.filename = self.url = base + '.webm'
+            self.dst_path = join(settings['destination'], path, base + '.webm')
```

Comment:
Good point.

Actually, I wonder how it should behave when it finds out a video not supported, like an avi file. So far it just, it just copies the file in the output. But when opening with a browser, the video can't be played.

I see here two options:
- Copy the file anyway, but provides a warning message in the output telling the file is not supported.
- Adding a new option, like `convert_unsupported` or similar, that performs a conversion on these cases. So the avi file would be converted in webm.